db/error-handler.js

```diff
@@ -0,0 +1,8 @@
+exports.errorHandler = (err, req, res, next) => {
+    if (err.status && err.msg) {
+        res.status(err.status).send({ msg: err.msg });
```

Comment:
Again you don't need this part yet inside the if block, nice job adding the 500 in the else though!